templates/taxi/manufacturer_list.html

```diff
@@ -21,30 +24,31 @@
       {% for manufacturer in manufacturer_list %}
         <tr>
           <td>
-              {{ manufacturer.id }}
+            {{ manufacturer.id }}
           </td>
           <td>
-              {{ manufacturer.name }}
+            {{ manufacturer.country }}
+            <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
+              Update
+            </a>
           </td>
           <td>
-              {{ manufacturer.country }}
+            {{ manufacturer.name }}
```

Comment:
The 'name' field is incorrectly placed in the 'Country' column. It should display 'manufacturer.country' instead.